Packages/com.unity.inputsystem/InputSystem/Monobehaviors/TrackedPoseDriver.cs

```diff
@@ -0,0 +1,237 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Experimental.Input;
+using UnityEngine.Experimental.Input.Controls;
+using UnityEngine.XR.PoseProvider;
+using UnityEngine.XR;
+
+namespace UnityEngine.Experimental.Input.XR

```

Comment:
Put in `UnityEngine.Experimental.Input.Plugins.XR`. Move file respectively.

Reply:
On it.